tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from pathlib import Path
+
+from .context import deploy
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--hub-name",
+        action="store",
+        default="staging"
+    )
+    parser.addoption(
+        "--cluster-name",
+        action="store",
+        default="2i2c"
+    )
+    parser.addoption(
+        "--api-token",
+        action="store",
+        default="secret"
+    )
+
+@pytest.fixture()
+def hub(request):
```

Comment:
Can you add a docstring about how this fixture is coming from the commandline params? This is a very awesome way to use fixtures!

Reply:
Sure <3